app/src/sharedTest/java/org/oppia/android/app/parser/ListItemLeadingMarginSpanTest.kt

```diff
@@ -535,13 +535,24 @@ class ListItemLeadingMarginSpanTest {
     val shadowCanvas = shadowOf(canvas)
 
     assertThat(shadowCanvas.textHistoryCount).isEqualTo(4)
+    // The below assertion verifies the 1st item of inner level list
     assertThat(shadowCanvas.getDrawnTextEvent(0).x).isEqualTo(72.0f)
```

Comment:
Ah I missed this before, but float checks should use ``isWithin`` with a tolerance, e.g.:

```kotlin
assertThat(textEvent0.x).isWithin(1e-5f).of(72.0f)
```

Ditto for all other float checks in this test suite.

Reply:
Done.